cvat/apps/auto_segmentation/views.py

```diff
@@ -59,19 +77,27 @@ def _convert_to_segmentation(mask):
     # Directory to save logs and trained model
     MODEL_DIR = os.path.join(ROOT_DIR, "logs")
 
+    if "mask" in model_path.lower() and not model_path.endswith("h5"):
+        model_path = os.path.join(model_path,"mask_rcnn_coco.h5")
+
     # Local path to trained weights file
-    COCO_MODEL_PATH = os.path.join(ROOT_DIR, "mask_rcnn_coco.h5")
+    COCO_MODEL_PATH = model_path
     if COCO_MODEL_PATH is None:
         raise OSError('Model path env not found in the system.')
     job = rq.get_current_job()
 
     ## CONFIGURATION
 
-    class InferenceConfig(coco.CocoConfig):
+    class InferenceConfig(CocoConfig):
         # Set batch size to 1 since we'll be running inference on
         # one image at a time. Batch size = GPU_COUNT * IMAGES_PER_GPU
         GPU_COUNT = 1
         IMAGES_PER_GPU = 1
+        NUM_CLASSES = num_c
+        local_device_protos = device_lib.list_local_devices()
+        num_gpus = len([x.name for x in local_device_protos if x.device_type == 'GPU'])
+        if num_gpus in [2,4,8]:
```

Comment:
`if num_gpus % 2 == 0`?

Reply:
Yeah, it can be changed. We only had 2,4 and 8 GPUs, so.

Reply:
I made the change.